mne/io/snirf/_snirf.py

```diff
@@ -221,6 +222,41 @@ def natural_keys(text):
                                                coord_frame=_frame_to_str[
                                                    coord_frame])
 
+            str_date = np.array((dat.get(
+                '/nirs/metaDataTags/MeasurementDate')))[0].decode('UTF-8')
+            str_time = np.array((dat.get(
+                '/nirs/metaDataTags/MeasurementTime')))[0].decode('UTF-8')
+            str_datetime = str_date + str_time
+
+            meas_date = None
+            # Several formats have been observed so we try each in turn
+            for dt_code in ['%Y-%m-%d%H:%M:%SZ',
+                            '%Y-%m-%d%H:%M:%S']:
+                try:
+                    meas_date = datetime.datetime.strptime(
+                        str_datetime, dt_code)
+                    meas_date = meas_date.replace(tzinfo=datetime.timezone.utc)
+                    break
+                except ValueError:
+                    pass
+            if meas_date is None:
+                warn("Extraction of measurement date from SNIRF file failed. "
+                     "The date is being set to January 1st, 2000, "
+                     "instead of {}".format(str_datetime))
```

Comment:
We're allowing use of f-strings now. Care to switch this to an f-string?

https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

Reply:
Nice idea. Thanks, old habits die hard